npm/src/auth/index.ts

```diff
@@ -33,6 +33,26 @@ const isRefreshTokenExpired: (
   return token != undefined && token.exp != undefined && token.exp < now
 }
 
+export const refreshIdleSessionTimeout: (
```

Comment:
Just thinking about the name for this function, because it sounds like it refreshes the token immediately. It would be good to make it more obvious that it starts a loop that repeatedly keeps the token refreshed.

What about something like `scheduleTokenRefresh` or `keepTokenUpdated`?